tools/docs/manager.ts

```diff
@@ -1,8 +1,20 @@
+import { DateTime } from 'luxon';
 import type { RenovateConfig } from '../../lib/config/types';
+import { logger } from '../../lib/logger';
 import { getManagers } from '../../lib/modules/manager';
+import { GithubHttp } from '../../lib/util/http/github';
+import { getQueryString } from '../../lib/util/url';
 import { readFile, updateFile } from '../utils';
+import type { GithubApiQueryResponse, ItemsEntity } from './github-query-items';
 import { getDisplayName, getNameWithUrl, replaceContent } from './utils';
 
+const gitHubApiUrl = 'https://api.github.com/search/issues?';
```

Comment:
```suggestion
const gitHubApiUrl = 'https://api.github.com/search/issues?';

if (process.env.GITHUB_TOKEN) {
  hostRules.Add({ matchHost: 'api.github.com', token: process.env.GITHUB_TOKEN });
}
```

Something like this should do the trick. That way the `GithubHttp` should automatically use the token.